chef/data_bags/crowbar/template-crowbar.schema

```diff
@@ -23,6 +23,15 @@
                 }
               }
             },
+            "client-users": { "type": "map", "required": true, "mapping": {
```

Comment:
I think we generally avoid `-` in the attribute names (because it prevents a number of magic things in ruby).

Btw, any reason that it's a list? Do we anticipate we may need more than one user?

Reply:
> I think we generally avoid - in the attribute names 

Will fix

> Btw, any reason that it's a list?

Just consistency with the other user list.